.github/workflows/prerelease-create-cloudflare.yml

```diff
@@ -0,0 +1,46 @@
+name: Prerelease
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Could you add `repository_dispatch` to make this testable?

Reply:
Done